components/ContractTokens.vue

```diff
@@ -61,7 +66,9 @@ import {
 const addresses: Record<string, string> = require("@/addresses.json")
 const contracts = namespace("contracts")
 
-@Component({ components: { MintTokens, RedeemTokens, ExpireContract } })
+@Component({
+  components: { MintTokens, SettleTokens, RedeemTokens, ExpireContract },
```

Comment:
So we actually don't need to import components because of the auto import feature in nuxt.config.js. So if they are in the components directory, they are are auto loaded and useable in the html template.

Reply:
Good to know. I removed the component imports